packages/test_packages_common.py

```diff
@@ -59,9 +79,7 @@ def test_import(name, selenium_standalone):
             )
         )
 
-    built_packages = _parse_package_subset(os.environ.get("PYODIDE_PACKAGES"))
-    # only a subset of packages were built
-    if built_packages is not None and name not in built_packages:
+    if name not in built_packages():
```

Comment:
It's more robust to actually check the artifacts present on disk instead of relying on the env variable, as we often pass it only to the make script and not to pytest.